xml/containers-podman-overview.xml

```diff
@@ -0,0 +1,152 @@
+    create and run containers. This means that &podman; can run in the root
+    mode as well as in an unpriviledged environment.
+   </para>
+  </listitem>
+ </itemizedlist>
+ <sect1 xml:id="cha-podman-install">
+  <title>&podman; Installation</title>
+
+  <para>
+   To install &podman;, run the <command>sudo zypper in podman</command>. Run
+   then <command>podman --version</command> to check whether &podman; has been
+   installed successfully.
+  </para>
+
+  <para>
+   By default, &podman; requires root privileges. To enable rootless mode for
+   the current user, run the following command:
+  </para>
+
+<screen>&prompt.user; sudo usermod --add-subuids 200000-201000 --add-subgids
+   200000-201000 $USER</screen>
+
+  <para>
+   Log out and log in to apply changes.
+  </para>
+
+  <para>
+   In certain situations, running a container with &podman; in the rootless
+   mode on &sls; may fail, because the container needs read access to the &scc;
+   credentials. For example, running a container with the <command>podman run
+   -it --rm registry.suse.com/suse/sle15 bash</command>, and then executing
+   <command>zypper ref</command> results in the following error message:
+  </para>
+
+<screen>Refreshing service 'container-suseconnect-zypp'.
+Problem retrieving the repository index file for service 'container-suseconnect-zypp':
+[container-suseconnect-zypp|file:/usr/lib/zypp/plugins/services/container-suseconnect-zypp] 
+Warning: Skipping service 'container-suseconnect-zypp' because of the above error.
+Warning: There are no enabled repositories defined.
+Use 'zypper addrepo' or 'zypper modifyrepo' commands to add or enable repositories</screen>
+
+  <para>
+   To solve the problem, grant the current user the required access rights by
+   running the following command on the host:
+  </para>
+
+<screen>&prompt.user;sudo setfacl -m u:$USER:r /etc/zypp/credentials.d/*</screen>
+
+  <para>
+   Log out and log in again to apply the changes.
+  </para>
+
+  <para>
+   To give multiple users the required access, create a dedicated group using
+   the <command>groupadd <replaceable>GROUPNAME</replaceable></command>
+   command. Use then the following command to change the group ownership and
+   rights of files in the <filename>/etc/zypp/credentials.d/</filename>
+   directory.
+  </para>
+
+<screen>&prompt.user;sudo chgrp <replaceable>GROUPNAME</replaceable> /etc/zypp/credentials.d/*
+&prompt.user;sudo chmod g+r /etc/zypp/credentials.d/*</screen>
+
+  <para>
+   You can then grant a specific user write access by adding them to the
+   created group.
+  </para>
+ </sect1>
+ <sect1 xml:id="cha-podman-use">
+  <title>&podman; Basic Usage</title>
+
+  <para>
+   Since &podman; is compatible with &deng;, it features the same commands and
+   options. For example, the <command>podman pull</command> command fetches a
+   container image from a registry, while the <command>podman build</command>
+   command is used to build images.
+  </para>
```

Comment:
Might this be a good place to suggest `alias docker=podman` ?

Reply:
I think we should not do that. There are enough smaller difference that just cause surprises/frustation if the `docker` command suddenly behaves weird.